showcase-v1alpha2/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "repository": "googleapis/nodejs-showcase",
+  "name": "showcase",
+  "version": "0.1.0",
+  "author": "Google LLC",
+  "description": "GAPIC Showcase API client for Node.js",
+  "main": "build/src/index.js",
+  "files": [
+    "protos",
+    "src",
+    "AUTHORS",
+    "COPYING"
+  ],
+  "keywords": [
+    "google apis client",
+    "google api client",
+    "google apis",
+    "google api",
+    "google",
+    "google cloud platform",
+    "google cloud",
+    "cloud",
+    "google showcase",
+    "showcase",
+    "GAPIC Showcase API"
+  ],
+  "dependencies": {
+    "google-gax": "googleapis/gax-nodejs#typescript-microgenerator",
+    "lodash.merge": "^4.6.0"
+  },
+  "devDependencies": {
+    "@types/duplexify": "^3.5.0",
+    "@types/lodash.merge": "^4.6.4",
+    "gts": "^0.8.0",
+    "mocha": "^5.2.0",
+    "through2": "^2.0.3",
+    "typescript": "~2.8.0"
+  },
+  "scripts": {
+    "publish-module": "node ../../scripts/publish.js showcase",
+    "smoke-test": "mocha smoke-test/*.js --timeout 5000",
+    "test": "mocha test/*.js",
```

Comment:
Assuming this should be `mocha build/test`

Reply:
Yes, I have not done anything for tests yet.